py/private/py_binary.bzl

```diff
@@ -73,11 +73,6 @@ def _py_binary_rule_impl(ctx):
             "{{ARG_COLLISION_STRATEGY}}": ctx.attr.package_collisions,
             "{{ARG_PYTHON}}": to_rlocation_path(ctx, py_toolchain.python) if py_toolchain.runfiles_interpreter else py_toolchain.python.path,
             "{{ARG_VENV_NAME}}": ".{}.venv".format(ctx.attr.name),
-            "{{ARG_VENV_PYTHON_VERSION}}": "{}.{}.{}".format(
```

Comment:
I don't remember why this was needed before. Is it desirable that we no longer need to tell it the python version? Or is this undesirable and now something can drift?

Reply:
The UV crate query Python before creating the venv for version and a bunch of other info, and as the venv is created at runtime (JIT style), we no longer need to pass this, nothing will drift.
It's still needed for unpacking a wheel however, as we don't want to run the interpreter as part of the action.